vivisect/analysis/arm/emulation.py

```diff
@@ -175,8 +182,33 @@ def buildFunctionApi(vw, fva, emu, emumon):
     return api
 
 
+def getAllReads(emu):
+    # there has to be a more efficient way...
```

Comment:
Probably. Maybe we should talk about deprecating the whole pathcore/graphcore stuff (and just straight up removing the dbcore stuff)

Reply:
you've mentioned that before.  i'm not a fan.  why
 would we?  i use it, don't you?
i mean, i guess we can kill dbcore stuff.  it seams like a precursor to Synapse, tbh.  i haven't used it, although that doesn't mean i might not in the near future.  i'm doing some potentially heavy-duty graph stuff.

is it broken?  is it costing?  i don't think it's ugly.

my point in this comment is that i think there may be a more efficient way to do what i'm doing here.  i'm not complaining about pathing in general.  if anything, i think pathcore/graphcore would benefit from a bit *more* code to make some of the powerful stuff more easily accessible.